src/Application/Common/Interfaces/ICrpgDbContext.cs

```diff
@@ -22,6 +23,7 @@ public interface ICrpgDbContext
     DbSet<EquippedItem> EquippedItems { get; }
     DbSet<CharacterLimitations> CharacterLimitations { get; }
     DbSet<Restriction> Restrictions { get; }
+    DbSet<Leaderboard> Leaderboard { get; }
```

Comment:
You don't need to save the leaderboard. As long as the rating is already computed, it's just a matter of ordering and limiting a query.

Reply:
so you want me to add a competitive rating property to characters and compute it regularly through a worker?
(remember , i did not want it to be computed in the mod as when the competitive rating formula would change we would get rating from different version mixed in the leaderboard)

Reply:
also i thought we agreed on doing it the way i did it , or there is something i did not understand 
![image](https://github.com/verdie-g/crpg/assets/47090987/c4e9135b-affb-406c-98a8-41ebbfe8e935)

Reply:
I was thinking about a worker that would update the rating. But thinking about it again it sounds a bit overkill just to solve the issue when you update the formula. I think a db migration whenever you update the formula would be way more simple.

Reply:
Ok. Let's say i move competitive rating to rating. Instead of doing migration everytime the formula change , can i add a PUT endpoint that would update the competitive rating. So instead of crafting migration and messing with db  , i could have a command called UpdateCompetitiveRating that i could call from the api.  Would that violate the Restful Philosophy?

 
![image](https://github.com/verdie-g/crpg/assets/47090987/89169f60-28d7-4328-88cd-b3a4ab87f688)

Reply:
Both are okay to me.

Reply:
done !